.github/workflows/sandbox.yaml

```diff
@@ -0,0 +1,23 @@
+name: sandbox
+
+#TODO - check if we can somehow get over GitHub limitation of only one 'on' per workflow
+on:
+  push:
+    paths:
+      - devkit/sandbox/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      #We need this to be checked out here as well
+      - name: Checkout the repository
+        uses: actions/checkout@master
```

Comment:
can we lock down the version used here with `actions/checkout@v3`?

Reply:
yes